src/main/java/io/orkes/conductor/client/WorkflowClient.java

```diff
@@ -68,6 +68,19 @@ public abstract BulkResponse terminateWorkflowsWithFailure(List<String> workflow
     public abstract Map<String, List<Workflow>> getWorkflowsByNamesAndCorrelationIds(
             List<String> correlationIds, List<String> workflowNames, Boolean includeClosed, Boolean includeTasks);
 
+    /**
```

Comment:
this is going to be super confusing with methods returning EXACT same return value.

Reply:
not if we mark the old one as `@Deprecated`

Reply:
@v1r3n How do you feel about moving ahead with this, given that we're deprecating the above method?

Reply:
let's do it.